tests/unittests/OperatorTest.cpp

```diff
@@ -1126,9 +1126,11 @@ TEST_P(OperatorTest, QuantizedTopK) {
   EXPECT_EQ(IH.at({2, 0, 2}), 4);
 }
 
-TEST_P(OperatorTest, Gather64) {
-  ENABLED_BACKENDS(Interpreter, CPU);
-
+/// Helper for testing Gather with different \p ITy / \p IndexType.
+template <typename IndexType>
+static void gatherFloatInputTest(glow::PlaceholderBindings &bindings_,
```

Comment:
I think the convention is to user terminal underscores for private variables within the scope of a class, not for function parameter names. Consider removing them.